composer.json

```diff
@@ -26,7 +26,7 @@
         "doctrine/doctrine-bundle": "^2.5",
         "doctrine/persistence": "^2.1 || ^3.0",
         "sonata-project/admin-bundle": "^4.15",
-        "sonata-project/block-bundle": "^4.16.2",
```

Comment:
It'll be fixed after this PR https://github.com/sonata-project/SonataBlockBundle/pull/1115 pass

Reply:
But I need this PR :/

Reply:
See https://github.com/sonata-project/SonataBlockBundle/pull/1115#issuecomment-1254704463